lace/serialization/obj/__init__.py

```diff
@@ -85,10 +85,17 @@ def path_relative_to_mesh(filename):
             with s3.open(materials_filepath, 'r') as f:
                 mesh.materials_file = f.readlines()
             mesh.materials_filepath = materials_filepath
+
     if hasattr(mesh, 'materials_file'):
-        for line in mesh.materials_file:
-            if line and line.split() and line.split()[0] == 'map_Ka':
-                mesh.texture_filepath = path_relative_to_mesh(line.split()[1])
+        mesh.materials = dict([
+            (line.split()[0], ' '.join(line.split()[1:]))
+            for line in mesh.materials_file if line.split()
+        ])
```

Comment:
```py
mesh.materials = dict([line.split(None, 1) for line line in mesh.materials_file if line.startswith('map_K')])
```
or
```py
mesh.materials = {line.split()[0]: line.split(None, 1)[1] for line line in mesh.materials_file if line.startswith('map_K')}
```
`split` takes two arguments, the delimiter to split on and the number of splits. We say `None` for the delimiter in order to get the default `split()` behavior of splitting on all whitespace and say we want one split (i.e. one cut into two sections). 
```py
>>> ' 1  2   3  '.split(None, 1)
['1', '2   3  ']
```
Since we know that `split(None, 1)` is going to return a 2 item list, the first option uses dict's list of tuples behavior to make those into key and value. The latter form is a little clearer, but requires calling split twice; dict comprehensions are just about always better than `dict()` on a list comprehension producing tuples.

The other change here is to `if` on the line starting with one of the texture map commands, which saves us a lot of calls to split and is more explicit anyway.